01week/datatypes.js

```diff
@@ -1,7 +1,9 @@
-//1.Current Day and Time
+
```

Comment:
Do you have your code in another pull request?

Reply:
I had the code in another branch, sorry. I created a new branch for it and a pull request. https://github.com/AustinCodingAcademy/javascript-workbook/pull/534